packages/next-on-pages/src/buildApplication/buildVercelOutput.ts

```diff
@@ -1,16 +1,11 @@
 import { writeFile, mkdir, rm, rmdir } from 'fs/promises';
 import { spawn, type ChildProcessWithoutNullStreams } from 'child_process';
 import { join, resolve } from 'path';
-import { cliLog, cliWarn } from '../cli';
+import { cliError, cliLog, cliWarn } from '../cli';
 import { readJsonFile, validateDir, validateFile } from '../utils';
-import type { PackageManager } from './packageManagerUtils';
-import {
-	getCurrentPackageManager,
-	getExecStr,
-	getPackageManagerInfo,
-	getPackageVersion,
-	waitForProcessToClose,
-} from './packageManagerUtils';
+import { getPackageManager } from 'package-manager-manager';
+import type { PackageManager } from 'package-manager-manager';
```

Comment:
NIT: feels like these imports should be higher up the file now?

Reply:
yeah thanks I'll move them 🙂

(as a followup we can add the eslint rule for the imports ordering 🤔)

Reply:
done :)